proto/sdk/alpha/alpha.proto

```diff
@@ -207,3 +247,62 @@ message UpdateCounterRequest {
   // (e.g. `count`, `capacity`; *not* `Counter.count` or `Counter.capacity`).
   google.protobuf.FieldMask update_mask = 2 [(google.api.field_behavior) = REQUIRED];
 }
+
+// A representation of a List.
+message List {
+  option (google.api.resource) = {
+      type: "agones.dev/List"
+      pattern: "lists/{list}"
+  };
+  // The name of the List
+  string name = 1;
+  // The maximum capacity of the List
+  optional int64 capacity = 2;
+  // The array of items in the List ["v1", "v2", …]
+  repeated string values = 3;
+}
+
+message GetListRequest {
+  // The name of the List to get
+  string name = 1 [
+                  (google.api.field_behavior) = REQUIRED,
+                  (google.api.resource_reference) = {
+    type: "agones.dev/List"
+  }];
+}
+
+message UpdateListRequest {
+  // The List to update
+  List list = 1 [
+                  (google.api.field_behavior) = REQUIRED,
+                  (google.api.resource_reference) = {
+    type: "agones.dev/List"
+  }];
+
+  // Required. Mask (list) of fields to update.
+  // Fields are specified relative to the List
+  // (e.g. `capacity`, `values`; *not* `List.capacity` or `List.values`).
+  google.protobuf.FieldMask update_mask = 2 [(google.api.field_behavior) = REQUIRED];
+}
+
+message AddListValueRequest {
+  // The name of the List to add a value to.
+  string name = 1 [
```

Comment:
I chose to leave it as "name" since I thought it would be confusing to have a string list, string name (GetListRequest), and a List list (UpdateListRequest).